### ISSUE TITLE
`listenActiveWindow` sends notification upon mouse move in current active window

### ISSUE THREAD

**Issue by kdouglass-oliveai** [open-olive/loop-development-kit#156]
`listenActiveWindow` should only send a notification when the focused window changes, not when your mouse moves within the active window.

https://user-images.githubusercontent.com/83777472/117330911-2d01fe00-ae64-11eb-8597-76508aef1793.mov

**Comment by cshields-oliveai** [comment]
@kdouglass-oliveai - No this is not intended behavior - I will bring this one back to the team and create a new defect for this.

Thanks for reporting!

**Comment by richseviora** [comment]
After some troubleshooting it looks like what's happening is that whenever a user is in Chrome and they're hovering over a link, the "active" window is changing from the Chrome window (with the page title and all that) to some other Google Chrome window:
```
2021-05-07T10:56:28-07:00 INF Received Event {"title":"Google Chrome","path":"Google Chrome","pid":2650,"x":-1886,"y":2107,"width":536,"height":22} console=info loopID=def system=secureloops
```
Actual Chrome window entry:
```
2021-05-07T10:56:30-07:00 INF Received Event {"title":"`listenActiveWindow` sends notification upon mouse move in current active window · Issue #156 · open-olive/loop-development-kit","path":"Google Chrome","pid":2650,"x":-1885,"y":25,"width":1896,"height":2103} console=info loopID=def system=secureloops
```
If I had to guess, Google Chrome is creating a window of sorts whenever a link has been hovered over, a dramatically smaller window (536x22) than the actual window (1896x2103).

**Comment by richseviora** [comment]
@kdouglass-oliveai A fix is now in code review for this issue. I will let you know when it's scheduled for release.

**Comment by kdouglass-oliveai** [comment]
Great, thanks @richseviora

**Comment by richseviora** [comment]
@kdouglass-oliveai , after a lot of internal debate we've decided not to suppress these events b/c there may be use cases for accessing it. Instead, we'll add some documentation that explains that some apps will generate multiple windows, and that Loop authors should consider the windows changed to/from.

**Comment by richseviora** [comment]
Will be fixed in #167

**Comment by richseviora** [comment]
Closing with documentation update
